README.md

```diff
@@ -107,6 +107,29 @@ memory model, and *any future memory model*. We ensure this by:
 [Miri]: https://github.com/rust-lang/miri
 [Kani]: https://github.com/model-checking/kani
 
+## Relationship to Project Safe Transmute
+
+[Project Safe Transmute] is an official initiative of the Rust Project to
+develop language-level support for safer transmutation. The Project consults
+with crates like zerocopy to identify aspects of safer transmutation that
+would benefit from compiler support, and has developed an [experimental,
+compiler-supported analysis][mcp-transmutability] of when a value of one
+type is soundly transmutable into another. Once this functionality is
+sufficiently mature, zerocopy intends to replace its internal
+transmutability analysis with the compiler-supported one. This change will
+likely be an implementation detail that is invisible to zerocopy's users.
+
+However, Project Safe Transmute will not replace the need for most of
```

Comment:
Given the last sentence of the previous paragraph, it probably makes sense to remove "However".